src/core/utils.js

```diff
@@ -243,21 +243,11 @@ class Utils extends UtilsCore {
 	 *
 	 */
 	static isArray(array) {
-		switch (array.constructor) {
-			case Int8Array:
-			case Uint8Array:
-			case Uint8ClampedArray:
-			case Int16Array:
-			case Uint16Array:
-			case Int32Array:
-			case Uint32Array:
-			case Float32Array:
-			case Float64Array:
-			case Array:
-				return true;
-			default:
-				return false;
+		if((array.length || array.length === 0) && (array.constructor !== String)) {
```

Comment:
What do you think about keeping this existing functionality, and adding a method that allows them to add their array type, that could iterate through, and if it exists we could return true.  That way it is all strongly typed.